Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Batch.java

```diff
@@ -78,9 +78,7 @@
      * The field processes holds the processes that belong to the batch.
      */
     @ManyToMany(cascade = CascadeType.PERSIST, fetch = FetchType.EAGER)
-    @JoinTable(name = "batch_x_process", joinColumns = {
-            @JoinColumn(name = "batch_id", foreignKey = @ForeignKey(name = "FK_batch_x_process_batch_id")) }, inverseJoinColumns = {
-                    @JoinColumn(name = "process_id", foreignKey = @ForeignKey(name = "FK_batch_x_process_process_id")) })
+    @JoinTable(name = "batch_x_process", joinColumns = {@JoinColumn(name = "batch_id", foreignKey = @ForeignKey(name = "FK_batch_x_process_batch_id")) }, inverseJoinColumns = {@JoinColumn(name = "process_id", foreignKey = @ForeignKey(name = "FK_batch_x_process_process_id")) })
```

Comment:
are you really sure that your code formatter works correctly? This line for sure breaks the length limit.

Reply:
I did not change anything else, so this file was probably never formatted with the current formatter. Maybe this is still a thing that needs fixing, in the Kitodo - Data Access code there are no such long annotations, so I don’t know why this happens.

Reply:
So probably again it is problem of relative length of lines in Eclipse formatter...